.github/openwrt/test.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+
+case "$1" in
+	"n2n-edge"|"n2n-supernode")
+		eval "$1" | grep "$2"
```

Comment:
> This doesnt look like a very useful test. It would probably make more sense to check the exit code of running the binary.

I have no idea. Do a actual runtime test can be very complex. So just do a version check here instead. If the binrary is broken it will also fail.